Mathlib/Data/Finmap.lean

```diff
@@ -72,7 +72,7 @@ structure Finmap (β : α → Type v) : Type max u v where
 def AList.toFinmap (s : AList β) : Finmap β :=
   ⟨s.entries, s.nodupKeys⟩
 
-local notation:arg "⟦" a "⟧" => AList.toFinmap a
+local notation:arg (priority := high) "⟦" a "⟧" => AList.toFinmap a
```

Comment:
For posterity, can you say why this is needed?

Reply:
I think that this was another one of these `choice` disambiguations.

Reply:
Thanks; what do you think about a comment here?